app/views/que/view/jobs/index.html.erb

```diff
@@ -51,7 +51,7 @@
         <td><%= job[:queue] %></td>
         <td><%= job[:run_at].strftime("%Y-%m-%d %H:%M:%S") %></td>
         <td>
-          <% job.dig(:args, 0, :arguments).each do |argument| %>
+          <% Array(job.dig(:args, 0, :arguments)).each do |argument| %>
```

Comment:
@saturnflyer thank you for found such edge case, but is it better to use just `&.each` ?

Reply:
Ideally the object would always return something that responds to `each`
I would rather not have coercion nor nil checking in the view and just have the view be handed an object that has the expected behavior.

I would have chosen that path but there are no tests to lean on for introducing an object for the view that does what the template needs to 

But iterating over an empty array results in no operation.

Reply:
ok, right now it's nice to have this fix, later I'll add tests and modify this